src/modules/configs/templates/final.py

```diff
@@ -1,34 +1,55 @@
+# All fields are required!
```

Comment:
All those comments need to be removed after creation of schema for final template.